src/Parts/MyTrainingCard/index.jsx

```diff
@@ -1,23 +1,23 @@
-import { Badge, Carousel, Col, Row } from 'antd';
-import { LeftOutlined, RightOutlined } from '@ant-design/icons';
-import { SingleTrainingCard } from './SingleTrainingCard';
-import './MyTrainingCard.css';
-import { useContext } from 'react';
-import { AppContext } from '../../Context';
+import { Badge, Carousel, Col, Row } from "antd";
+import { LeftOutlined, RightOutlined } from "@ant-design/icons";
+import { SingleTrainingCard } from "./SingleTrainingCard";
+import "./MyTrainingCard.css";
+import { useContext } from "react";
+import { AppContext } from "../../Context";
 
-export const MyTrainingCard = () => {
+const MyTrainingCard = () => {
   const { MyTrainingCardContext } = useContext(AppContext);
   return (
     <>
```

Comment:
Change empty tag with react fragment

Reply:
thankyou for review, this issue already fix in a7711523a75ebf1bee4167f348adc6eda02d0c44